app/models/solidus_paypal_commerce_platform/paypal_address.rb

```diff
@@ -23,7 +25,11 @@ def update(paypal_address)
 
     private
 
-    def format_address(address)
+    def add_email_to_order(recipient)
+      @order.update(email: recipient[:email_address])
```

Comment:
Currently, this does return false if the update fails. It does not block the address from being added to the order, because email isn't part of the address validation. 

But I think this is fine because this situation only occurs when the user is a guest - if they were logged in, we would already have a valid email address. So if the order doesn't have a valid email address, when this completes and the user is directed to the checkout path, they get the registration page - asking for them to either log in or give a valid email. Does that work for you, or do you think we need to throw an alert or something on the page that they are trying to check out from?

![image](https://user-images.githubusercontent.com/5720486/86596179-6bd2d900-bf5f-11ea-9cec-93330a6469e8.png)